src/test/http-mocks/claim-store.ts

```diff
@@ -94,7 +94,7 @@ export function resolveRetrieveClaimByClaimNumber (claimOverride?: object): mock
 
 export function rejectRetrieveClaimByExternalIdWithNotFound (reason: string) {
   mock(`${serviceBaseURL}/claims`)
-    .get(new RegExp('/[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}'))
+    .get(new RegExp('/[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$'))
```

Comment:
why not `^` at the beginning of the regEx?

Reply:
Only worried about terminating it, it was matching the wrong one in route tests